src/coreclr/jit/compiler.h

```diff
@@ -9943,6 +9943,23 @@ class Compiler
         return (compOpportunisticallyDependsOn(InstructionSet_EVEX));
     }
 
+    //------------------------------------------------------------------------
+    // canUseRex2Encoding - Answer the question: Is Rex2 encoding supported on this target.
+    //
+    // Returns:
+    //    `true` if Rex2 encoding is supported, `false` if not.
+    //
+    bool canUseRex2Encoding() const
+    {
+#ifdef DEBUG
+        if (JitConfig.JitBypassAPXCheck())
+        {
+            return true;
+        }
```

Comment:
This shouldn't be needed given the `JitStressApxCheck` right?

It can rather be handled via the general `AltJit` support for the "normal" path, like we do for other ISAs?